### PR TITLE
Added the target_status field to the bid obj

### PR DESCRIPTION
**Pull request by IvanNoveo**
Card: https://sprints.zoho.eu/team/redspherproduct#itemdetails/P20/I702

Solr update schema:

- bid
`curl -X POST -H 'Content-type:application/json' --user 'solr:yoctu' --data '{ "add-field": { "name": "target_status", "type": "strings", "uninvertible": false, "multiValued": true, "indexed": true, "stored": true, "required": false  } }' 'http://solr.test.yoctu.solutions:8983/api/collections/bid/schema'`

- bid-archive
`curl -X POST -H 'Content-type:application/json' --user 'solr:yoctu' --data '{ "add-field": { "name": "target_status", "type": "strings", "uninvertible": false, "multiValued": true, "indexed": true, "stored": true, "required": false  } }' 'http://solr.test.yoctu.solutions:8983/api/collections/bid-archive/schema'`

### CODE REVIEW

File: structures/bid.js [hunk 1]
```diff
@@ -62,7 +62,8 @@ exports.bid = function (config = null, auction = null) {
             deDateUtc: s.optional(dateUtc()),
             deDateRangeUtc: s.optional(dateUtc()),
             bid_score: s.optional(s.number()),
-            bid_score_process: s.optional(s.size(s.string(), 2, 254))
```
Comment (gbo-redspher) [inline structures/bid.js:65]: Can it only be `approved` ? I've asked @rendakbalazs confirmation

**Review by gbo-redspher — CHANGES_REQUESTED** [review]
wait for the enum validation before merging

**Review by gbo-redspher — APPROVED** [review]

**Comment by gbo-redspher** [comment]
@IvanNoveo Add the correct curl to update the solr in the description of this PR and in the one you'll do in shaq also

**Comment by IvanNoveo** [comment]
@gbo-redspher Done. I've also updated test Solr instance

**Review by gbo-redspher — APPROVED** [review]

**Pull request merged**